controllers/keda/configmap_controller.go

```diff
@@ -36,18 +36,19 @@ import (
 )
 
 const (
-	configMapName      = "keda-metrics-apiserver"
-	configMapNamespace = "keda"
+	configMapName = "keda-metrics-apiserver"
 )
 
 // ConfigMapReconciler reconciles a ConfigMap object
 type ConfigMapReconciler struct {
 	client.Client
-	Log    logr.Logger
-	Scheme *runtime.Scheme
+	Log                logr.Logger
+	Scheme             *runtime.Scheme
+	configMapNamespace string
```

Comment:
might be better to give this a better name, the configmap there is a bit confusing

Reply:
will do